app/notifications/followup_notification.rb

```diff
@@ -7,7 +7,8 @@ class FollowupNotification < BaseNotification
   # Add your delivery methods
   #
   deliver_by :database
-  # deliver_by :email, mailer: "UserMailer"
+  # deliver_by :email, mailer: "UserMailer", if: :email_notifications?
+  # deliver_by :sms, class: "DeliveryMethods::Sms", if: :sms_notifications?
```

Comment:
is this supposed to be commented out?

Reply:
Yes, the email one was commented out when I started this. https://github.com/rubyforgood/casa/pull/1612/files is the email and database pr

Reply:
Also the noticed gm was first added here https://github.com/rubyforgood/casa/pull/1611